recipes/clair3-trio/meta.yaml

```diff
@@ -22,21 +22,21 @@ requirements:
     - bzip2
     - xz
     - zlib
-    - python=3.6.10
+    - python 3.6.10
```

Comment:
Those pins look way to strict, are they really needed?

Reply:
yes, please merge.

Reply:
Agree with @bgruening that these pinnings are aggressively too strict

Reply:
We sought to alleviate the pinnings but the different behaviors of CUDA dependencies that were required differently by multiple TensorFlow versions left us with no other good options. In our new tools, we started using Pytorch which requires fewer pinnings, but for this tool, we'd prefer to stick to the pinnings for now.